package.json

```diff
@@ -1,14 +1,7 @@
 {
   "name": "twitter-fetcher",
-  "version": "17.1.0",
-  "keywords": [
-    "Twitter",
-    "OAUTH",
-    "JavaScript",
-    "tweets",
-    "api",
-    "widget"
-  ],
+  "version": "18.0.2",
```

Comment:
18.0.3

Reply:
ahaha, looks like they get their own numbers wrong! This is all coming directly from their merge

Reply:
Derp... Well as long as it gets fixed for our own release/tag, whatever that may be!